### DISCUSSION TITLE
Import genome assembly

### DISCUSSION THREAD

**Discussion by gibberwocky** [labsquare/CuteVariant-python#565] [Q&A] [answered]
I've just installed cutevariant but can't see a means to add a different genome assembly. I'm working on a variety of species, so would be useful to be able import non-human genomes as in IGV. If there's a way to do this can someone please advise?

Cheers

D

**Comment by dridk** [comment, marked as answer]
Actually the genome assembly input is only used for information purpose. You can put any word or skip it. 
Cutevariant should works for any diploïdes species. For non-diploïde, it may works but with strange behavior regarding genotypes fields. 

Thanks you very much for testing our software. I would appreciate any feedback from you.

**Reply by gibberwocky** [reply]
Thanks, I realised shortly after posting that I could proceed without selecting a genome.

**Reply by dridk** [reply]
Ok ! 
Please continue to give us your feedback about usability and feature enhancement !
